GSA/made-in-america
# [000] feature/prettier

## Description

better extends prettier and json config, updates config to use json for constistency

## How to test

- fetch changes locally
- `npm install`
- alter a json/ts file
- stage file
- commit file locally to let precommit hook run
- confirm that file gets prettified
- and linter passes

To prepare for code review, please follow the following checks.
